scripts/lc-builds/corona_sycl.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJAPerf/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+if [[ $# -lt 1 ]]; then
+  echo
+  echo "You must pass 1 argument to the script (in this order): "
+  echo "   1) SYCL compiler installation path"
+  echo
+  echo "For example: "
+  echo "    corona_sycl.sh /usr/workspace/raja-dev/clang_sycl_hip_gcc10.2.1_rocm5.1.0/install"
+  exit
+fi
+
+SYCL_PATH=$1
+shift 1
+
+BUILD_SUFFIX=corona-sycl
+: ${BUILD_TYPE:=RelWithDebInfo}
+RAJA_HOSTCONFIG=../tpl/RAJA/host-configs/lc-builds/toss4/corona_sycl.cmake
+
+echo
+echo "Creating build directory build_${BUILD_SUFFIX} and generating configuration in it"
+echo "Configuration extra arguments:"
+echo "   $@"
+echo
+
+rm -rf build_${BUILD_SUFFIX}_${USER} >/dev/null
+mkdir build_${BUILD_SUFFIX}_${USER} && cd build_${BUILD_SUFFIX}_${USER}
+
+DATE=$(printf '%(%Y-%m-%d)T\n' -1)
+
+export PATH=${SYCL_PATH}/bin:$PATH
+
+## NOTE: RAJA tests are turned off due to compilation issues.
+
+cmake \
+  -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+  -DSYCL_LIB_PATH:STRING="${SYCL_PATH}/lib" \
+  -C ${RAJA_HOSTCONFIG} \
+  -DENABLE_OPENMP=Off \
+  -DENABLE_CUDA=Off \
+  -DRAJA_ENABLE_TARGET_OPENMP=Off \
+  -DENABLE_ALL_WARNINGS=Off \
+  -DENABLE_SYCL=On \
+  -DCMAKE_C_COMPILER=clang \
+  -DCMAKE_CXX_COMPILER=clang++ \
+  -DCMAKE_LINKER=clang++ \
+  -DCMAKE_CXX_STANDARD=17 \
```

Comment:
What about BLT_CXX_STD?

Reply:
I believe CMAKE_CXX_STANDARD covers it. I did not change this script, merely copied it over from RAJA repo so we can use/modify it when we have an updated sycl compiler to use.